packages/css-library/config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
I converted this file to `js` from `json` format because future tickets have us adding custom token formats that will require [style-dictionary custom transforms like this](https://github.com/department-of-veterans-affairs/component-library/pull/766/files#diff-79ccf6c33f36bac9578b5a05b5b76e3b80ae31b308d1ee41f941bb8fd449ebb0).